node/child_process.ts

```diff
@@ -166,9 +165,12 @@ export function spawn(
   maybeOptions?: SpawnOptions,
 ): ChildProcess {
   const args = Array.isArray(argsOrOptions) ? argsOrOptions : [];
-  const options = !Array.isArray(argsOrOptions) && argsOrOptions != null
+  let options = !Array.isArray(argsOrOptions) && argsOrOptions != null
     ? argsOrOptions
-    : maybeOptions;
+    : maybeOptions as SpawnOptions;
+
+  options = normalizeSpawnArguments(command, args, options);
```

Comment:
Note: This sets `windowsVerbatimArguments=true` when the shell is cmd on windows. ref: https://github.com/denoland/deno_std/blob/dfbd08fd408cb4e6cbf1c664ce61c8dcc9b09b3f/node/internal/child_process.ts#L549-L551